README.md

```diff
@@ -1,11 +1,18 @@
-# custom-elements-json
-A file format for describing custom elements
+# custom-elements.json
+A file format for describing custom elements.
 
-# Status
+Its primary goal is to offer an overview of the full public API of multiple custom elements.
 
-This effort is in the initial discussion and requirements-gathering phase. Please join the discussion in [Issues](https://github.com/webcomponents/custom-elements-json/issues).
+Key points for web components are:
+- properties
+- attributes
+- events
+- methods
+- [slots](https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element) 
+- [CSS custom properties](https://drafts.csswg.org/css-variables/#defining-variables)
```

Comment:
Is "CSS custom variables" the more correct name?

Reply:
> A custom property is any property whose name starts with two dashes

I think the term is correct... I always say css variables and then correct myself to css custom properties as I think that is the spec name... but I could be totally wrong 🙈

Reply:
I'm being super picky, but you asked :) "Its primary goal is to offer...."
"Its" feels awkward here, given how short the doc is you're likely talking about custom-elements.json, but you could also mean this document maybe. I'd be more direct and say "The primary goal of custom-elements.json is to offer..."